docs/user-guide/rug/generators.md

```diff
@@ -16,46 +16,135 @@ Let's look more closely at what makes a project a Rug generator.
+    └── .travis.yml
 ```
 
-To enable this project as a Rug generator, we simply add a `.atomist`
-directory to the root of the project.
+To turn our project into a [Rug project][rugproj], we can use the 
+`ConvertExistingProjectToGenerator` Rug generator in [rug-editors][convert] to 
+add all the necessary directories and files:
+
+[rugproj]: projects.md
+[convert]: https://github.com/atomist-rugs/rug-editors#convertexistingprojecttogenerator
 
 ```console
-mkdocs-project
-    ├── .atomist
-    │   ├── manifest.yml
-    │   ├── package.json
-    │   ├── editors
-    │   │   └── NewMkDocsDocumentationProject.ts
-    │   └── tests
-    │       └── NewMkDocsDocumentationProject.rt
-    ├── README.md
-    ├── docs
-    │   └── index.md
-    ├── mkdocs.yml
-    └── requirements.txt
+$ rug edit atomist-rugs:rug-editors:ConvertExistingProjectToGenerator \
+    archive_name=spring-boot-service \
+    group_id=com.company.rugs \
+    version=0.13.0 \
+    generator_name=NewSpringBootService \
+    description="Rug project for Spring Rest Services"
+Processing dependencies                                                                                                                                                                                          
+  Downloading atomist-rugs/rug-editors/maven-metadata.xml ← rugs (740 bytes) succeeded                                                                                                                 
+  Downloading atomist-rugs/rug-editors/maven-metadata.xml ← global (740 bytes) succeeded                                                                                                               
+  Downloading atomist-rugs/rug-editors/0.14.0/rug-editors-0.14.0.pom ← rugs (635 bytes) succeeded                                                                                                      
+  Downloading atomist-rugs/rug-editors/0.14.0/rug-editors-0.14.0-metadata.json ← rugs (14 kb) succeeded                                                                                                
+  Downloading atomist-rugs/rug-editors/0.14.0/rug-editors-0.14.0.zip ← rugs (194 kb) succeeded                                                                                                         
+Resolving dependencies for atomist-rugs:rug-editors:latest completed
+Loading atomist-rugs:rug-editors:0.14.0 into runtime completed
+  TypeScript files added, run `cd .atomist && npm install`                                                                                                                                                      
+                                                                                                                                                                                                                
+Running editor ConvertExistingProjectToGenerator of atomist-rugs:rug-editors:0.14.0 completed
+
+→ Project
+  ~/dev/oss/atomist-docs-samples/ (14 kb in 20 files)
```

Comment:
Most of your examples use `~/workspace` as the working directory. We should probably be consistent.